src/Assignment4/SearchComponent/index.js

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+
+import "./styles.css";
+
+const SearchComponent = ({ setCurrentCards, teamInformation }) => {
+  const [nameOrDesignation, setNameOrDesignation] = useState("");
+  return (
+    <form
+      onSubmit={(event) => {
```

Comment:
Rathe than writing this long piece of code to handle on submit you can also make a function like handleSubmit and attach it here, just a suggestion as it Looks good and more readable.

Reply:
sure, i generally do that. will push it in my next PR.